portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -23,25 +23,42 @@
 import java.util.ArrayList;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
-@WebServlet("/data")
+// @WebServlet("/data")
```

Comment:
The website now logs a 404 error saying it tried and failed to send a GET request to /data. Is that important?

Reply:
That is to be expected right now because I am not using it, I am looking at probably getting rid of it altogether